plugins/resource_monitor_plugin/resource_monitor_plugin.cpp

```diff
@@ -139,7 +147,11 @@ class resource_monitor_plugin_impl {
    static constexpr uint32_t space_threshold_high = 99; // in percentage
    static constexpr uint32_t space_threshold_warning_diff = 5; // Warning issued when space used reached (threshold - space_threshold_warning_diff). space_threshold_warning_diff must be smaller than space_threshold_low
 
-   boost::asio::io_context     ctx;
+   static constexpr uint32_t def_monitor_warning_interval = 30; // After this number of monitor intervals, warning is output if the threshold is hit
+   static constexpr uint32_t warning_interval_low = 1;
+   static constexpr uint32_t warning_interval_high = 450; // e.g. if the monitor interval is 2 sec, the warning interval is at most 15 minutes
```

Comment:
consider using min and max in names

Reply:
Thanks. I changed all `_low` and `high` in the limits names to `_min` and `_max`.